frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx

```diff
@@ -264,7 +256,7 @@ export const PopinFixedHeader = (props) => {
                 label={action.label}
                 key={action.label}
                 onClick={action.onClick} // eslint-disable-line react/jsx-handler-names
-                customClass='transparentButton'
+                customClass={action.separatorLine ? 'transparentButton dropdownMenuSeparatorLine' : 'transparentButton'}
```

Comment:
use the library classnames to generate dynamic class strings

Reply:
done